libwget/ssl_wolfssl.c

```diff
@@ -30,6 +30,12 @@
  * https://test-sspev.verisign.com:2443/test-SSPEV-revoked-verisign.html
  *
  */
+#ifndef DEBUG_WOLFSSL
+#include <wolfssl/options.h>
```

Comment:
In L54, `wolfssl/options.h` is included a second time. Can you move the code there ?